Paper.tex

```diff
@@ -452,19 +452,19 @@ \subsubsection{Block Header Validity}
 
 This mechanism enforces a homeostasis in terms of the time between blocks; a smaller period between the last two blocks results in an increase in the difficulty level and thus additional computation required, lengthening the likely next period. Conversely, if the period is too large, the difficulty, and expected time to the next block, is reduced.
 
-The nonce, $H_n$, must satisfy the relation:
+The nonce, $H_n$, must satisfy the relations:
 \begin{equation}
-%\mathtt{PoW}(H, H_n) \dfrac{\leqslant}{den} \frac{2^{256}}{H_d}
-\mathtt{PoW}(H, H_n) \leqslant \frac{2^{256}}{H_d}
+\mathtt{PoW}(H_{\hcancel{n}}, H_n, \mathbf{d})[0] = H_m \quad \wedge \quad \mathtt{PoW}(H_{\hcancel{n}}, H_n, \mathbf{d})[1] \leqslant \frac{2^{256}}{H_d}
 \end{equation}
 
-Where $\mathtt{PoW}$ is the proof-of-work function (see section \ref{ch:pow}): this evaluates to an pseudo-random number cryptographically dependent on the parameters $H$ and $H_n$. Given an approximately uniform distribution in the range $[0, 2^{256})$, the expected time to find a solution is proportional to the difficulty, $H_d$.
+Where $H_{\hcancel{n}}$ is the new block's header $H$, but \textit{without} the nonce component, $\mathbf{d}$ being the current DAG, a large data set needed to compute the mixHash, and $\mathtt{PoW}$ is the proof-of-work function (see section \ref{ch:pow}): this evaluates to an array with the first item being the mixHash, to proof that a correct DAG has been used, and the second item being a pseudo-random number cryptographically dependent on $H$ and $\mathbf{d}$. Given an approximately uniform distribution in the range $[0, 2^{64})$, the expected time to find a solution is proportional to the difficulty, $H_d$.
 
 This is the foundation of the security of the blockchain and is the fundamental reason why a malicious node cannot propagate newly created blocks that would otherwise overwrite (``rewrite'') history. Because the nonce must satisfy this requirement, and because its satisfaction depends on the contents of the block and in turn its composed transactions, creating new, valid, blocks is difficult and, over time, requires approximately the total compute power of the trustworthy portion of the mining peers.
 
 Thus we are able to define the block header validity function $V(H)$:
 \begin{eqnarray}
-V(H) & \equiv & \mathtt{PoW}(H, H_n) \leqslant \frac{2^{256}}{H_d} \quad \wedge \\
+V(H) & \equiv & \mathtt{PoW}(H_{\hcancel{n}}, H_n, \mathbf{d})[0] = H_m \quad \wedge \\
```

Comment:
Subscripting to a named component through use of `[` and numeral index is breaking convention with the rest of the paper. For consistency, define the evaluated value to be a mapping and from two named components (e.g. m, n).

Thus:

```
n \leqslant \frac{2^{256}}{H_d} \wedge m = H_m
where (n, m) = \mathtt{PoW}(H_{\hcancel{n}}, H_n, \mathbf{d})
```

Of course `PoW` function will have to be altered accordingly.